docs/targets/polkadot.rst

```diff
@@ -62,3 +61,45 @@ The following example shows how call flags can be used:
 .. include:: ../examples/polkadot/call_flags.sol
   :code: solidity
 
+
+Reverts and error data decoding
+_______________________________
+
+When a contract reverts, the returned error data is what
+`EVM would return <https://docs.soliditylang.org/en/v0.8.20/control-structures.html#panic-via-assert-and-error-via-require>`_.
+``assert()``, ``require()``, or ``revert()`` will revert the contract execution, where the revert reason 
+is supplied as the contracts output (if any). Solidity contracts can also revert with a `Panic` 
+(please refer to the 
+`Solidity language documentation <https://docs.soliditylang.org/en/v0.8.20/control-structures.html#panic-via-assert-and-error-via-require>`_.
```

Comment:
Please call this `Ethereum Solidity documentation. It's totally confusing to call this "Solidity language documentation".

Reply:
Why is it confusing calling the Solidity Language Documentation the Solidity Language Documentation.

If anything, `Ethereum Solidity documentation` is confusing. What is `Ethereum Solidity`? What is not `Ethereum Solidity`?

I don't get why we can't call the Solidity Language by it's name.

You go to a website, the website is called "https://soliditylang.org/".

This implies that there is a thing called "Solidity Language".

On this website, there is a button, called "read the docs", leading to another website at "https://docs.soliditylang.org".

So, if there is the `Solidity Langauge`, and it has a `Documentation`. How is it "totally confusing" to calling it "Solidity language documentation"?